### PR TITLE
implemented nft mint and erc1155 and login with eth

### PR DESCRIPTION
**Pull request by unicorn-talent**
NFT mint with erc1155
Fixed dialog issues
Login with Ethereum (Metamask, Portis, Walletconnect, Fortmatic)

### CODE REVIEW

File: src/pages/SignIn.tsx [hunk 1]
```diff
@@ -1,25 +1,124 @@
...
-
+import { selectUser, setUser } from "../store/user/userSlice";
+const { VITE_FORT_MATIC_KEY, VITE_PORTIS_KEY, VITE_WALLETCONNECT_KEY } =
+  import.meta.env;
 export default function SignIn(): ReactElement {
   const [user] = useAuthState(auth);
   const navigate = useNavigate();
-
+  const [account, setAccount] = useState("");
+  const [connection, setConnection] = useState<any>(false);
```
Comment (kmozurkewich) [inline src/pages/SignIn.tsx]: @unicorn-talent

File: src/pages/SignIn.tsx [hunk 2]
```diff
@@ -1,25 +1,124 @@
...
+  const [loggedIn, setLoggedIn] = useState<Boolean>(false);
```
Comment (kmozurkewich) [inline src/pages/SignIn.tsx]: @unicorn-talent

**Review by kmozurkewich — CHANGES_REQUESTED** [review]
looks really good! can you remove the dead code mentioned by the linters?

File: src/pages/SignIn.tsx (hunk 2 above)
Comment (unicorn-talent) [reply src/pages/SignIn.tsx]: its been fixed.

**Review by unicorn-talent — COMMENTED** [review]
fixed link warnings

**Review by nickroberts — APPROVED** [review]

**Review by nickroberts — APPROVED** [review]

**Comment by nickroberts** [comment]
This was deployed, and the site did not load. Not sure what went wrong, but this was in the console:

```
ReferenceError: Can't find variable: require
```

**Pull request merged**